src/components/Toast.vue

```diff
@@ -87,7 +87,7 @@ export default {
     max-width: 300px;
     opacity: 0;
     z-index: 1;
-    border-radius: $border-radius;
+    border-radius: var(--border-radius);
```

Comment:
I like this as well, except that I think `--border-radius` should be set per theme and not via settings